pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -1193,6 +1193,41 @@ func (in Nodes) Any(f func(node NodeStatus) bool) bool {
 	return false
 }
 
+// Children returns the children of the parent.
+func (s Nodes) Children(parentNodeId string) Nodes {
```

Comment:
I think this is the best place for those 3 functions.

Reply:
nice